### ISSUE TITLE
transmissions.send arguments question

### ISSUE THREAD

**Issue by jasonrhodes** [SparkPost/node-sparkpost#125]
What possible siblings are there to `transmissionBody` in the `sparkpost.transmissions.send()` argument? If none, can we remove that layer in version 2.x and just set options like:

```js
var sp = new SparkPost(<api key>);

sp.transmissions.send({
  content: {},
  recipients: []
}, callback);
```

?

**Comment by aydrian** [comment]
I'm for it. I was going to suggest something like this. Let's make the nodejs library great again.

**Comment by aydrian** [comment]
Closed by #158
